### ISSUE TITLE
file_finder plugin finds incorrect files current directory is not repo root

### ISSUE THREAD

**Issue by jaraco** [jaraco/jaraco.vcs#9]
Originally reported by: **Jason R. Coombs (BitBucket: [jaraco](http://bitbucket.org/jaraco), GitHub: [jaraco](http://github.com/jaraco))**

---

I recently encountered an error when trying to pip install an hgtools-enabled-package while residing in a mercurial repository. pip creates a subdirectory for installing the package, but since that directory has a parent which is a mercurial repository, hgtools incorrectly finds files in that parent repo, which causes setuptools to ignore the manifest:

```
        rcfiles = list(walk_revctrl())
        if rcfiles:
            self.filelist.extend(rcfiles)
        elif os.path.exists(self.manifest):
            self.read_manifest()
```

Therefore, hgtools file finder plugin needs to be smarter about only returning files relative to the directory it was supplied, regardless of were the root of the repo is.

---
- Bitbucket: https://bitbucket.org/jaraco/hgtools/issue/9

**Comment by jaraco** [comment]
_Original comment by_: **Jason R. Coombs (BitBucket: [jaraco](http://bitbucket.org/jaraco), GitHub: [jaraco](http://github.com/jaraco))**

---

This is the same issue reported in the parent project: https://bitbucket.org/jezdez/setuptools_hg/issue/3

**Comment by jaraco** [comment]
_Original comment by_: **Jason R. Coombs (BitBucket: [jaraco](http://bitbucket.org/jaraco), GitHub: [jaraco](http://github.com/jaraco))**

---

Removing version: 1.1 (automated comment)

**Comment by jaraco** [comment]
_Original comment by_: **Jason R. Coombs (BitBucket: [jaraco](http://bitbucket.org/jaraco), GitHub: [jaraco](http://github.com/jaraco))**

---

SubprocessManager now only returns those files found relative to the specified location. Fixes #9.
